tests/core/evaluation/test_marker_tracker_loader.py

```diff
@@ -20,6 +21,36 @@ def marker_trackerstore() -> TrackerStore:
     return store
 
 
+@pytest.fixture
+def multi_session_tracker() -> TrackerStore:
+    """Sets up a TrackerStore with 1 tracker with 2 sessions."""
+    temp_db = "temp.db"
+    domain = Domain.empty()
+    store = SQLTrackerStore(domain, db=temp_db)
+    tracker = DialogueStateTracker("test123", None)
+    tracker.update_with_events(
+        [
+            UserUttered("0"),
+            UserUttered("1"),
+            SessionStarted(),
+            UserUttered("2"),
+            UserUttered("3"),
+        ],
+        domain,
+    )
+    store.save(tracker)
+    yield store
+    os.remove(temp_db)
+
+
+def test_load_sessions(multi_session_tracker):
```

Comment:
(Type was missing here -- but we should move the stuff that is in the fixture into this function anyway :)